appliances/backup_and_restore.md

```diff
@@ -15,6 +15,22 @@ Creating a backup of the appliance will be done using SSH or the console.
 
 3. Save and/or rename `/root/backup.tgz`
 
+# Scheduling Database Backups
+Scheduling a database backup using crontab and pg_dump
+
+1. SSH into the appliance and edit the root user crontab `crontab -e`
+2. Press `i` to insert and add a line with the desired schedule to run the backup job. Examples:
+   ```bash
+     # Every hour on the hour
+     0 * * * * pg_dump -Fc vmdb_production > /tmp/db_dump$(date +\%Y\%m\%dT\%H\%M\%S).dump
```

Comment:
We may want to add a note about tmp space and/or attaching disks or something to that effect (or is that already in the backup docs?)

Reply:
Done